src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2107,6 +2107,20 @@ For more information, see https://github.com/ow2-proactive/scheduling/blob/maste
 === Catalog
 include::./Catalog.adoc[]
 
+[[_backup]]
```

Comment:
I would put this section on 3.4 Server backup mechanism

Reply:
ok